test/global_test_setup.clj

```diff
@@ -0,0 +1,8 @@
+(ns global-test-setup
+  "This namespace's contents will be automatically loaded by `lein test` (or any test runner)."
+  (:require
+   [refactor-nrepl.inlined-deps.toolsnamespace.v1v1v0.clojure.tools.namespace.repl :refer [set-refresh-dirs]]))
```

Comment:
Ideally this would be:

```diff
- refactor-nrepl.inlined-deps.toolsnamespace.v1v1v0.clojure.tools.namespace.repl
+ clojure.tools.namespace.repl
```

but I couldn't wait get it to work. It looked like mranderson :expositions would do the trick but nope?

Reply:
It seems like this old issue to me https://github.com/benedekfazekas/mranderson/issues/5